test/lib/ansible_test/_util/controller/sanity/code-smell/runtime-metadata.py

```diff
@@ -221,7 +255,7 @@ def validate_metadata_file(path, is_ansible, check_deprecation_dates=False):
 
     import_redirection_schema = Any(
         Schema({
-            ('redirect'): Any(*string_types),
+            ('redirect'): Any(fqcr, collection_name),
```

Comment:
This one should stay *string_types too, I think. From a quick test it seems like import_redirection can redirect to anything, not just collection content.

Reply:
:+1: implemented in ef78fbd9b767101617c7529bc876fb4b3bda66ba

Not touching module_utils / import redirects in this PR is probably best for now.